app/serializers/api/order_serializer.rb

```diff
@@ -23,10 +23,6 @@ def changes_allowed_until
       I18n.l(object.order_cycle.andand.orders_close_at, format: "%b %d, %Y %H:%M")
     end
 
-    def total
-      object.total.to_money.to_s
-    end
-
```

Comment:
we don't use it anymore?

Reply:
Yes we do but it would still appear as it's declared in the [`attributes` method](https://github.com/ltrls/openfoodnetwork/blob/c43e40a4c460fb5f7ae352f44a5d5fbc3aec8238/app/serializers/api/order_serializer.rb#L3). What we want is that it's like `def total; object.total; end` and there's no need to have this as it's like not writing the method (customizing the field) at all.